k8s/openebs-pre-release-features.yaml

```diff
@@ -25,14 +25,15 @@ spec:
   # will be placed
   - name: RunNamespace
     value: openebs
-  # ResourceLimits allow you to set the limits on memory and cpu for pool pods
+  # PoolResourceLimits allow you to set the limits on memory and cpu for pool pods
```

Comment:
If `PoolResourceLimits` and `AuxResourceLimits` can be specified from SPC, what is intention to put it here?
Additionally if these are not found in SPC as well as CAS Template configs then it will be false by default.

Reply:
By default, there wont be any limits. Hence setting it to false inside the CAS templates. This feature is mostly for users to specify depending on their environment. 

There is no need to specify in SPC or StorageClasses and we can just document it. But I thought adding them there will serve as a reminder for users who might actually skip reading the documentation.

Reply:
Giving those in comments is fine,`PoolResourceLimits` is commented but not `AuxResourceLimits`

Reply:
I see. That should be fixed!